learn/configuration/instance_options.md

```diff
@@ -371,14 +371,18 @@ Sets the maximum size of [accepted payloads](/learn/core_concepts/documents.md#d
 
 ### Schedule snapshot creation
 
-::: warning
-🚩 This option does not take any values. Assigning a value will throw an error. 🚩
-:::
-
 **Environment variable**: `MEILI_SCHEDULE_SNAPSHOT`
 **CLI option**: `--schedule-snapshot`
+**Default value**: disabled if not present, `86400` if present without a value
+**Expected value**: `None` or an integer
+
+Activates scheduled snapshots. Snapshots are disabled by default.
+
+It is possible to use `--schedule-snapshot` without a value. If `--schedule-snapshot` is present when launching an instance but has not been assigned a value, Meilisearch takes a new snapshot every 24 hours.
 
-Activates scheduled snapshots when provided. Snapshots are disabled by default.
+For more control over snapshot scheduling, pass an integer representing the interval in seconds between each snapshot. When `--schedule-snapshot=3600`, Meilisearch takes a new snapshot every hour.
+
+When using the configuration file, it is also possible to explicitly pass a boolean value to `schedule_snapshot`. Meilisearch takes a new snapshot every 24 hours when `schedule_snapshot=true`, and takes no snapshots when `schedule_snapshot=false`.
```

Comment:
Is it a good idea to have this in a container so it stands out to the reader?
::: note Scheduling snapshots in the configuration file (or something shorter)
:::

Reply:
It's not a bad idea. It helps breaking up the wall of text this option turned out to be.